lib/__tests__/standalone.test.js

```diff
@@ -107,6 +107,81 @@ it("standalone with non-existent-file and allowEmptyInput false should throw err
   })
 })
 
+it("standalone with two non-existent-files and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/non-existent-file.css and ${fixturesPath}/another-non-existent-file.css does not match any files`)
```

Comment:
If we're using `x and y` syntax we also need to change the verb from `does` to `do`. I'd be fine simplifying it by reversing the syntax: `Could not find files matching x and y`.